charts/dex/templates/ingress.yaml

```diff
@@ -1,6 +1,9 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "dex.fullname" . -}}
 {{- $svcPort := .Values.service.ports.http.port -}}
+{{- if .Values.https.enabled -}}
+{{- $svcPort = .Values.service.ports.https.port -}}
```

Comment:
It seems like a breaking change to me because users must explicitly tell their ingress controllers that the backend serves HTTPS, e.g., by providing the `nginx.ingress.kubernetes.io/backend-protocol: HTTPS` annotation.

Such changes are beyond patch releases.